Provide XML Schema files for portlet-model-hints.xml and custom-sql/default.xml

dmarks2 opened the pull request:
The plugin can provide XML Schema for XML files which do not have a schema definition. By this pull request I would like to contribute XML Schema definitions for portlet-model-hints.xml and custom-sql/default.xml. 
By providing the XML Schema code completion is possible in Model Hints and Custom SQL XML files, too.

jtydhr88 commented:
hey @gamerson,
I can help @dmarks2 to do source format works

gamerson commented:
Thanks Terry!

On Wed, May 30, 2018 at 9:21 PM, Terry Jia <notifications@github.com> wrote:

> hey @gamerson <https://github.com/gamerson>,
> I can help @dmarks2 <https://github.com/dmarks2> to do source format works
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gamerson/liferay-intellij-plugin/pull/5#issuecomment-393380540>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAkVFe_W4wAtgvSqICZJH8XH5ceGAC-Bks5t31OzgaJpZM4UQ8Qp>
> .
>



-- 
Greg Amerson
Liferay Backend Developer Tools
Liferay, Inc. www.liferay.com

dmarks2 commented:
Hey @gamerson,

I added the copyright, changed the email address of the commit user and added a --signoff. I changed the source code, so that the formatSource gradle command does not show any more warnings.

gamerson commented:
Pull request submitted to https://github.com/gamerson/liferay-intellij-plugin/pull/6. [See changes here.](https://github.com/gamerson/liferay-intellij-plugin/compare/943f8af~2...943f8af) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
